### ISSUE TITLE
[Proposal]: `ignore-same` on Weblate

### ISSUE THREAD

**Issue by AntimatterReactor** [naev/naev#2557]
### Suggestion

After encountering same translation check fails for the 42nd time due to contributors' name being untranslatable, I feel like that specific check is more annoying for what it's worth. So in my humble opinion I think we should just ignore it using `ignore-same`, although I might be wrong---maybe there are sane reasons why same translations shouldn't be ignored---in which case just ignore my ramblings.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by UncombedCoconut** [comment]
No, you're right, I just wasn't seeing the misguided warnings. I've added "ignore-same".

**Comment by AntimatterReactor** [comment]
Ah, ok then, thanks!
